spec/calendar_assistant/cli/repl_spec.rb

```diff
@@ -0,0 +1,93 @@
+describe CalendarAssistant::CLI::Repl do
+  class FakeReadLine
+    def initialize(inputs)
+      @inputs = Array(inputs)
+    end
+
+    def readline(*args)
+      return nil if @inputs.empty?
+      @inputs.pop
+    end
+  end
+
+  describe CalendarAssistant::CLI::Repl::History do
+    with_temp_file("history")
+
+    before do
+      @current_history_file = ENV['THOR_HISTORY_FILE']
+      ENV['THOR_HISTORY_FILE'] = temp_file.path
```

Comment:
Making this a real config param as suggested above simplifies these tests by allowing injection in code and not env vars.